server.js

```diff
@@ -2,36 +2,125 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import netflixData from './data/netflix-titles.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
+const Show = mongoose.model('Show', {
+  title: { type: String },
+  director: { type: String },
+  cast: { type: String },
+  country: { type: String },
+  date_added: { type: String },
+  release_year: { type: Number },
+  rating: { type: String },
+  duration: { type: String },
+  listed_in: { type: String },
+  description: { type: String },
+  type: { type: String }
+})
+
+const Director = mongoose.model('Director', {
+  director: String,
+  title: {
+    type: mongoose.Schema.Types.String,
+    ref: 'Title'
+  }
+})
+
+const Title = mongoose.model('Title', {
+  title: String,
+  _id: Number,
```

Comment:
Mongoose adds unique _id field so it is no need to add one by hand. This will make your work harder when you will start adding new records to the database later.